lib/mixlib/install/generator/bourne/scripts/platform_detection.sh

```diff
@@ -46,8 +46,13 @@ elif test -f "/etc/Eos-release"; then
 elif test -f "/etc/redhat-release"; then
   platform=`sed 's/^\(.\+\) release.*/\1/' /etc/redhat-release | tr '[A-Z]' '[a-z]'`
   platform_version=`sed 's/^.\+ release \([.0-9]\+\).*/\1/' /etc/redhat-release`
+  
+  if test "$platform" = "rocky linux"; then
+  	source /etc/os-release
+ 	os="${ROCKY_SUPPORT_PRODUCT}"
```

Comment:
can we instead use `"${REDHAT_SUPPORT_PRODUCT}"` here as `"${ROCKY_SUPPORT_PRODUCT}"` will include version too, which we are separately fetching.